guide/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "devDependencies": {
     "chexo": "1.0.7",
-    "hexo": "7.3.0",
+    "hexo": "3.9.0",
```

Comment:
😨

Reply:
Originally, this dependency was set to 3.9.0 for the guide app, as it is compatible. With version 7.3.0, significant incompatibilities arise. I mistakenly upgraded to 7.3.0 in release 3.1.2, and that is the reason of the difference in this branch.

![image](https://github.com/user-attachments/assets/f333c1d5-986b-4258-854d-f36d2211147c)